packages/components/src/components/Dropdown/Dropdown.tsx

```diff
@@ -31,7 +31,6 @@ const Container = styled.div<{ disabled?: boolean }>`
     all: unset;
     width: fit-content;
     height: fit-content;
-    border-radius: 50%;
```

Comment:
why is this removed?

Reply:
it broke labels for example

Reply:
It was making the clickable area of the default toggle component match that component's shape:

<img width="84" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/ffd449c8-a53d-4f6b-913a-1ca586307488">
<img width="84" alt="image" src="https://github.com/trezor/trezor-suite/assets/45338719/96a453e7-4d57-4b00-a9cc-b72e740952d4">

So perhaps it should be present if the default toggle is there, at least? 🤔

Reply:
Can you send me the place in the app where I can see this? I am not sure if I understand. Example would help. Thanks! :)

Reply:
Any place where Dropdown uses the original toggle. And actually removing this introduces a bug, that was a reason why I put is there. Usually it's a good practice to ask a person who added the style when you see one you do not understand. 🙏 But even better to write the comment.

https://github.com/trezor/trezor-suite/assets/45338719/d571e02d-bf78-4ba9-bddd-808a3f8c0e1e

Reply:
Git it. Problem was it was just WTF, it was a hack with no comment to explain it.

This is what it does when you use custom children and not default `(...)`
![image](https://github.com/trezor/trezor-suite/assets/152899911/4098ac41-64e4-443d-8e23-61d5339c84d2)

Reply:
I will prepare PR with proper fix.

Reply:
https://github.com/trezor/trezor-suite/pull/11107